### PR TITLE
Add support for playback controller command issued events

### PR DESCRIPTION
**Pull request by rmberrios**
Add support for playback controller command issued events

### CODE REVIEW

File: lib/StateMachineSkill.js [hunk 1]
```diff
@@ -34,14 +34,28 @@ class StateMachineSkill extends AlexaSkill {
       return this.requestHandlers.IntentRequest(alexaEvent, reply);
```
Comment (armonge) [inline lib/StateMachineSkill.js]: why have this as a separate piece? maybe we should also put the LaunchRequest handling with the rest of `requestToTurnIntoIntent`

Comment (rmberrios) [reply lib/StateMachineSkill.js]: Done!

File: lib/StateMachineSkill.js [hunk 2]
```diff
@@ -43,12 +35,18 @@ class StateMachineSkill extends AlexaSkill {
...
-          .drop(2)
-          .join('')
-          .replace('.', '')
-          .value();
+        let intent;
+
+        if (requestToConvert === 'onLaunchRequest') {
+          intent = this.config.openIntent || 'LaunchIntent';
+        } else {
+          intent = _.chain(requestToConvert)
```
Comment (armonge) [inline lib/StateMachineSkill.js]: this is also something hard to ready, maybe we could split it into a separate function? Like, `requestNameToIntent` or something like that is easier to understand than
```js
            .split('')
            .drop(2)
            .join('')
            .replace('.', '')
            .value();
```

We could also have a test that shows `onPlaybackController.PlayCommandIssued` is transformed into `PlaybackControllerPlayCommandIssuedIntent` will make us understand why the need for all those 'replace', drop, etc

Comment (rmberrios) [reply lib/StateMachineSkill.js]: Done!

**Pull request merged**